packages/dbgpt-serve/src/dbgpt_serve/file/tests/test_fsspec_impl.py

```
@@ -0,0 +1,30 @@
+    client.storage_system.get_file_metadata.return_value = object()
+    client.delete_file_by_id.return_value = True
+    filesystem = DBGPTFileSystem(client=client)
+
+    filesystem.rm("documents/file.txt")
+
+    client.delete_file_by_id.assert_called_once_with("documents", "file.txt")
+
+
+@pytest.mark.parametrize(
+    "path",
+    ["/tmp/folder/file.txt", "file:///tmp/folder/file.txt"],
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/folder/file.txt"

(S108)